Console_File_Maneger/ConsoleUserInerface.cs

```diff
@@ -0,0 +1,186 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+using Library_File_Maneger;
+using Library_File_Maneger.User_Command;
+
+namespace Console_File_Maneger
+{
+    internal sealed class ConsoleUserInerface : IUserInterface
+    {
+        DataDirectores[] DataDirs;
+
+        public ConsoleUserInerface(DataDirectores[] DataDirs)
+        {
+            DisplayConsole.Print();
+            DisplayConsole.PrintDisplay2Line();
+            this.DataDirs = DataDirs;
+        }
+
+        //============ Print ========================
+        public void PrintLeftWindow(string[] str)
+        {
+            Print(str, DataDirs[0].Page, 0);
+        }
+        public void PrintRightWindow(string[] str)
+        {
+            Print(str, DataDirs[1].Page, 1);
+        }
+        public void PrintInfoWindow(string[] str)
+        {
+            int lint_y = 1;
+            foreach (var line in str)
+            {
+                DisplayConsole.SetCursorPosition(2, DisplayConsole.Line1_2 + lint_y++);
+                DisplayConsole.PrintWrite(line);
+            }
+        }
+        public void PrintCommandWindow(string str)
+        {
+            DisplayConsole.PrintDirCommandLine(str);
+        }
+        public void PrintPages(int now, int max)
+        {
+            DisplayConsole.ChangrForegroundColor(Color.Blue);
+            string pages = "╣ Page " + (now + 1) + "/" + max + " ╠";
```

Comment:
Конкатенация строк - фу-фу-фу!
```suggestion
            string pages = $"╣ Page {now + 1}/{max} ╠";
```